frontend/packages/integration-tests-cypress/tests/crud/namespace-crud.spec.ts

```diff
@@ -53,54 +54,113 @@ describe('Namespace', () => {
 
   it('nav and breadcrumbs restores last selected "All Projects" when navigating from details to list view', () => {
     nav.sidenav.clickNavLink(['Workloads', 'Secrets']);
+    listPage.rows.shouldBeLoaded();
     projectDropdown.selectProject('All Projects');
     projectDropdown.shouldContain('All Projects');
     cy.log(
       'Nav from list page to details page should change Project from "All Projects" to resource specific project',
     );
-    cy.get(`[data-test-rows="resource-row"]`)
-      .first()
-      .find('a')
-      .first()
-      .click();
+    listPage.rows.clickFirstLinkInFirstRow();
+    detailsPage.isLoaded();
     projectDropdown.shouldNotContain('All Projects'); // after drilldown to Details page, project should be specific to resource
```

Comment:
This isn't new, but this should be "All Namespaces" if the tests are running against native k8s.

Reply:
Not new, but in each of these cases, it's better to check that the project dropdown has the expected project name. (Here and in the new tests below.)

Reply:
Hi @spadgett, switched to clicking on `alertmanager-main` in order to specifically test for `openshift-monitoring` project/namespace.  Not sure if that helps with the k8s native support though : /